editor/src/core/model/element-metadata-utils.spec.ts

```diff
@@ -490,26 +470,26 @@ describe('getElementLabel', () => {
     [TP.toString(spanElementMetadata.templatePath)]: spanElementMetadata,
     [TP.toString(divElementMetadata.templatePath)]: divElementMetadata,
   }
-  const components: Array<ComponentMetadata> = [
-    {
-      scenePath: scenePath,
-      templatePath: instancePath,
-      component: 'App',
-      sceneResizesContent: false,
-      rootElements: [divElementMetadata.templatePath],
-      globalFrame: canvasRectangle({
-        x: 0,
-        y: 0,
-        width: 100,
-        height: 100,
-      }),
-      style: {
-        width: 100,
-        height: 100,
-      },
-    },
-  ]
-  const metadata = jsxMetadata(components, elements)
+  // const components: Array<ComponentMetadata> = [
```

Comment:
Delete me?

Reply:
Oops :grinning:

Reply:
Oh yeah, this was deleted in a later commit, but that commit was moved into #1040, so this has been deleted :)